dvc/command/metrics.py

```diff
@@ -12,7 +12,46 @@ def _show(self, metrics):
                 logger.info("{}:".format(branch))
 
             for fname, metric in val.items():
-                logger.info("\t{}: {}".format(fname, metric))
+                self._output_formatted_metric(fname, metric)
+
+    def _is_json(self, content):
+        import json
+
+        try:
+            json.loads(content)
+            return True
+        except ValueError:
+            return False
+
+    def _output_formatted_metric(self, fname, metric):
+        import csv
+
+        if "\n" not in metric:
```

Comment:
I think that metric could be an array when `xpath` option is provided and "\n" could be potentially an element in that array?

Reply:
It could be @shcheklein , but I think it is a rare case -- I thought metrics have a numeric nature, maybe I'm a little bit bias and don't taking into account everything.